### PR TITLE
chore: 🍰 Fix Cypress Tests Unreliability

### LINKED ISSUE #2243: 🐛 [Bug] Cypress tests are not reliable to run on any system yet
## 🐛 Bugreport
- wait time for URLs and elements is set to short
- some selectors rely on language dependent identifiers
- script command (`"cypress"`) overwrites option to use Cypress locally with its GUI (`cypress open`).

### PR DESCRIPTION
**Pull request by mahula**
## 🍰 Pullrequest
Fix reliability issues and an error related to missing files

### Issues
- fixes #2243

### Todo
- [X] set wait time for URLs and elements higher
- [X] fix language dependence of several selectors in login and user profile tests
- [X] change script command for Cypress execution so it does not overwrite the option to use Cypress locally with its GUI
- [X] add missing registration test files

### CODE REVIEW

File: e2e-tests/cypress/README.md [hunk 1]
```diff
@@ -1,24 +1,57 @@
...
+- [User profile - change password](https://github.com/gradido/gradido/blob/master/e2e-tests/cypress/tests/cypress/e2e/UserProfile.ChangePassword.feature)
+- [User registration]((https://github.com/gradido/gradido/blob/master/e2e-tests/cypress/tests/cypress/e2e/User.Registration.feature)) (WIP)
 
-A sample setup to show-case Cypress as an end-to-end testing tool for Gradido running in a Docker container.
-Here we have a simple UI-based happy path login test running against the DEV system.
 
 ## Precondition
-Since dependencies and configurations for Github Actions integration is not set up yet, please run in root directory
+
+Before running the test, change to the the repo's root directory (gradido) and boot up the system under test
```
Comment (Tirokk) [inline e2e-tests/cypress/README.md:17]:
```suggestion
Before running the test, change to the repo's root directory (gradido) and boot up the system under test:
```

I would put a colon here …

File: e2e-tests/cypress/README.md [hunk 2]
```diff
@@ -1,24 +1,57 @@
...
 ```bash
 docker-compose up
 ```
 
-to boot up the DEV system, before running the test.
 
 ## Execute the test
 
+This setup will be integrated in the Gradido Github Actions to automatically support the CI/CD process.
+For now the test setup can only be use locally in two modes
```
Comment (Tirokk) [inline e2e-tests/cypress/README.md:27]:
```suggestion
For now the test setup can only be used locally in two modes.
```

File: e2e-tests/cypress/README.md [hunk 3]
```diff
@@ -1,24 +1,57 @@
...
+
+# install all dependencies
+yarn install
+
+# a) run the tests on command line
+yarn cypress run
+
+# b) open the Cypress GUI to run the tests in interactive mode
+yarn cypress open
+```
```
Comment (Tirokk) [inline e2e-tests/cypress/README.md:43]: I would add somehow that the database has to be seeded upfront to every test run.

**Review by Tirokk — CHANGES_REQUESTED** [review]
Hey cool! 💫

I recommend some little changes, see below.

Now it works if we seed the database upfront to every test run. 😍
That hint I would like to be added to the docu.
Do you agree?

And may we could have a hint where one finds the videos … ?

File: e2e-tests/cypress/README.md (hunk 3 above)
Comment (mahula) [reply e2e-tests/cypress/README.md:59]: Important point, than you.
With [f780b1c](https://github.com/gradido/gradido/commit/f780b1c431341cae1e075f8a97f88485cd407de9) the hint about seeding is added to the Readme.

File: e2e-tests/cypress/README.md [hunk 4]
```diff
@@ -1,24 +1,73 @@
...
+- [User profile - change password](https://github.com/gradido/gradido/blob/master/e2e-tests/cypress/tests/cypress/e2e/UserProfile.ChangePassword.feature)
+- [User registration]((https://github.com/gradido/gradido/blob/master/e2e-tests/cypress/tests/cypress/e2e/User.Registration.feature)) (WIP)
 
-A sample setup to show-case Cypress as an end-to-end testing tool for Gradido running in a Docker container.
-Here we have a simple UI-based happy path login test running against the DEV system.
 
 ## Precondition
-Since dependencies and configurations for Github Actions integration is not set up yet, please run in root directory
+
+Before running the test, change to the repo's root directory (gradido)
```
Comment (Tirokk) [inline e2e-tests/cypress/README.md:17]:
```suggestion
Before running the test, change to the repo's root directory (gradido).
```

**Review by Tirokk — DISMISSED** [review]
One very little suggestion …

File: e2e-tests/cypress/README.md (hunk 4 above)
Comment (mahula) [reply e2e-tests/cypress/README.md:17]: Sure, why not.

Comment (Tirokk) [reply e2e-tests/cypress/README.md]: Done

**Review by Tirokk — APPROVED** [review]
Hey, very good! 😍

A big step towards full testing! 🚀🚀🚀💫💫🌟

I approve!

**Pull request merged**